src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -291,11 +291,11 @@ private void checkIfGroupsExists(Course course) {
         // get all courses with exercises for this user
         List<Course> courses = courseService.findAllActiveWithExercisesAndLecturesForUser(user);
         Set<Exercise> activeExercises = courses.stream().flatMap(course -> course.getExercises().stream()).collect(Collectors.toSet());
-
         log.debug("          /courses/for-dashboard.findAllActiveWithExercisesForUser in " + (System.currentTimeMillis() - start) + "ms");
-        // TODO: can we only load the relevant result (the latest rated one which is displayed in the user interface)
-        List<StudentParticipation> participations = participationService.findWithResultsByStudentId(user.getId(), activeExercises);
-        log.debug("          /courses/for-dashboard.findWithResultsByStudentUsername in " + (System.currentTimeMillis() - start) + "ms");
+
+        long startParticipationCall = System.currentTimeMillis();
```

Comment:
Why do we log this here? Seems to be for debugging purposes and not part of this PR.

Reply:
I just *copied* it from the other service calls in the method. I'm not sure if you mean the log in general or just the specific start time via `startParticipationCall`, therefore I just removed the time and not the whole log.